charts/ziti-controller/values.yaml

```
@@ -95,6 +95,8 @@ ca:
   duration: 87840h # 3660d / 10 y
   # -- Go time.Duration string format
   renewBefore: 720h # 30d
+  # Set a custom cluster domain if other than cluster.local
```

```suggestion
  # -- Set a custom cluster domain if other than cluster.local
```

If you use this `# --` to indicate a comment on a property then `helm-docs` will automatically document that value in the README. :+1: